src/data/questions.ts

```diff
@@ -79,7 +78,7 @@ export const questions: Question[] = [
   ),
   q(
     'LIVE_APART_FROM_SPOUSE',
-    `Did you lived apart from your spouse for all of ${CURRENT_YEAR}?`,
+    `Did you lived apart from your spouse for all of the year?`,
```

Comment:
Should be _Have you lived apart from your spouse for the current tax year?_ rather than _Did you lived_

Reply:
Going with "did you live" because it's the period it's talking about doesn't include the present.